recipe/build.sh

```diff
@@ -12,14 +12,22 @@ case $target_platform in
         make install
         make check -j${CPU_COUNT}
         ;;
-    linux-*)
+    linux-64)
         ln -s `which $CC` $BUILD_PREFIX/bin/gcc
         export CC=$BUILD_PREFIX/bin/gcc
         ./configure --prefix=$PREFIX --enable-cblas --enable-threading=pthreads x86_64
         make CC_VENDOR=gcc -j${CPU_COUNT}
         make install
         make check -j${CPU_COUNT}
         ;;
+    linux-aarch64)
```

Comment:
Given that the only difference is a single variable here (for the arch), I doubt that this duplication is necessary...

Reply:
Well, `linux-64` needs `configure ... x86_64` and `linux-aarch64` needs `configure ... amd64`. 

I can add a conditional instead in a `linux-*` block but it feels it's a bit clearer like that (and there is a fair bit of repetition already between the linux and mac case so I thought it would be alright).

Reply:
So, I have refactored things. I'm expecting the current version to fail. We shall see.